### PR TITLE
Kibana buttons

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

**Comment by ahmadia** [comment]
RTM?

**Comment by brittainhard** [comment]
not yet

**Comment by ahmadia** [comment]
*hopping impatiently from one foot to the other*

**Comment by brittainhard** [comment]
okay, feel free to merge

**Comment by ahmadia** [comment]
are you sure?

File: source/base/templates/base/project.html [hunk 1]
```diff
@@ -41,7 +41,13 @@ <h4 style="line-height:2em;">
         <a href="{% url 'base:indices' project_slug=project.slug %}">
           <i class="fa fa-archive"> &nbsp; </i>Indices
         </a>
-        <a id="gotoKibana" disabled style="float:right;" class="btn btn-default btn-duo center link-button" href="/{{ project.slug}}/kibana">Goto Kibana</a>
+        <a id="gotoKibana"
+            {% if not settings.DEPLOYMENT %}
+            disabled
+            {% endif %}
+            style="float:right;" class="btn btn-default btn-duo center link-button" href="http://localhost:9999">
+            Goto Kibana
```
Comment (ahmadia) [inline source/base/templates/base/project.html]: Any reason not to just have the text here say "Kibana"?

**Comment by brittainhard** [comment]
yeah its good now

File: source/base/templates/base/project.html (hunk 1 above)
Comment (ahmadia) [inline source/base/templates/base/project.html]: The Goto thing seems a bit weird to me.

**Comment by ahmadia** [comment]
Woohoo.

**Pull request merged**